src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1067,17 +1067,17 @@
      *
      * <p>For example:</p>
      * <pre>
-     * |--this -&gt; 1
+     * this(1);
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * |--CTOR_CALL -&gt; CTOR_CALL
-     * |    |--LPAREN -&gt; (
-     * |    |--ELIST -&gt; ELIST
-     * |        |--EXPR -&gt; EXPR
-     * |            |`--NUM_INT -&gt; 1
-     * |    |--LPAREN -&gt; )
-     * |    |--SEMI -&gt; ;
+     * --CTOR_CALL -&gt; CTOR_CALL
```

Comment:
One `|` is still needed:
```
CTOR_CALL -> this
 |--LPAREN -> (
 |--ELIST -> ELIST
 |   `--EXPR -> EXPR
```
...

Reply:
Please guide me through the development IDE setup, the process given in documentation is a bit confusing.

Reply:
https://checkstyle.org/beginning_development.html